discord/gateway.py

```diff
@@ -166,7 +166,7 @@ def run(self) -> None:
             data = self.get_payload()
             _log.debug(self.msg, self.shard_id, data['d'])
             coro = self.ws.send_heartbeat(data)
-            f = asyncio.run_coroutine_threadsafe(coro, loop=self.ws.loop)
+            f = asyncio.run_coroutine_threadsafe(coro, self.ws.loop)
```

Comment:
What's the point of this change?

Reply:
This is also when I removed the `loop` attribute and then re-added it